packages/scrum-client/src/App.css

```diff
@@ -1,22 +1,12 @@
 @import url('https://fonts.googleapis.com/css?family=Fira+Sans:400,500,600,700,800');
 
-* {
-  box-sizing: border-box;
-}
-
-h1,h2,h3,h4,h5,h6,label,span {
-  font-weight: 500;
-  font-family: 'Fira Sans', sans-serif;
-}
-
 .App {
-  /* text-align: center; */
-  margin-top: 15px;
+  max-height: calc(100% - 120px);
+  overflow-y: scroll;
+  padding: 0px 10% !important;
```

Comment:
Why !important;? 
It usually shadows errors or bad levels of specifty on the css selector.

Reply:
is the same margin-top for all devices?

Reply:
Removed the important part, as i explained, this was more down to the lack of knowledge I had 
On the margins, i dont know, I may need some help on that and also regarding the scrolling. Ill create a issue and assign that to you if you can help me. I dont have much experience on css to the level you may have :D 

Issue #100 (https://github.com/carlospatinos/scrum/issues/100)